lib/content-services/src/lib/search/components/search-properties/search-properties.component.ts

```diff
@@ -0,0 +1,216 @@
+import { FormBuilder } from '@angular/forms';
+import { FileSizeCondition } from './file-size-condition';
+import { FileSizeOperator } from './file-size-operator.enum';
+import { FileSizeUnit } from './file-size-unit.enum';
+import { Subject } from 'rxjs';
+import { SearchWidgetSettings } from '../../models/search-widget-settings.interface';
+import { SearchQueryBuilderService } from '../../services/search-query-builder.service';
+import { SearchProperties } from './search-properties';
+import { TranslateService } from '@ngx-translate/core';
+import { SearchWidget } from '../../models/search-widget.interface';
+
+@Component({
+    selector: 'adf-search-properties',
+    templateUrl: './search-properties.component.html',
+    styleUrls: ['./search-properties.component.scss'],
+    encapsulation: ViewEncapsulation.None
+})
+export class SearchPropertiesComponent implements OnInit, AfterViewChecked, SearchWidget {
+    id: string;
+    settings?: SearchWidgetSettings;
+    context?: SearchQueryBuilderService;
+    startValue: SearchProperties;
+    displayValue$ = new Subject<string>();
+
+    private _form = this.formBuilder.nonNullable.group<FileSizeCondition>({
+        fileSizeOperator: FileSizeOperator.AT_LEAST,
+        fileSize: undefined,
+        fileSizeUnit: FileSizeUnit.KB
+    });
+    private _fileSizeOperators = Object.keys(FileSizeOperator).map<string>(key => FileSizeOperator[key]);
+    private _fileSizeUnits = [FileSizeUnit.KB, FileSizeUnit.MB, FileSizeUnit.GB];
+    private canvas = document.createElement('canvas');
+    private _fileSizeOperatorsMaxWidth: number;
+    private _selectedExtensions: string[];
+    private _reset$ = new Subject<void>();
+    private sizeField: string;
+    private nameField: string;
+
+    @ViewChild('fileSizeOperatorSelect', {read: ElementRef})
+    fileSizeOperatorSelectElement: ElementRef;
+
+    get form(): SearchPropertiesComponent['_form'] {
+        return this._form;
+    }
+
+    get fileSizeOperators(): string[] {
+        return this._fileSizeOperators;
+    }
+
+    get fileSizeUnits(): FileSizeUnit[] {
+        return this._fileSizeUnits;
+    }
+
+    get fileSizeOperatorsMaxWidth(): number {
+        return this._fileSizeOperatorsMaxWidth;
+    }
+
+    get reset$(): Subject<void> {
+        return this._reset$;
+    }
+
+    set selectedExtensions(extensions: string[]) {
+        this._selectedExtensions = extensions;
+    }
+
+    constructor(private formBuilder: FormBuilder, private translateService: TranslateService) {}
+
+    ngOnInit() {
+        if (this.settings) {
+            if (!this.settings.fileExtensions) {
+                this.settings.fileExtensions = [];
+            }
+            [this.sizeField, this.nameField] = this.settings.field.split(',');
+        }
+        if (this.startValue) {
+            this.setValue(this.startValue);
+        }
+    }
+
+    ngAfterViewChecked() {
```

Comment:
Unfortunately that is not possible because we can have different languages etc. Only way to do it properly and working for each language is calculate width of options and take the highest one. 
Solution which you suggested would work in case where we have only one language but that is not a case here.